dpgen/dispatcher/Slurm.py

```diff
@@ -103,6 +103,8 @@ def sub_script_head(self, res):
                 temp_exclude += ","
             temp_exclude = temp_exclude[:-1]
             ret += '#SBATCH --exclude=%s \n' % temp_exclude
+        for flag in res.get('custom_flags', []):
```

Comment:
@felix5572 Do you think it's better to use a `list` or a `dict`?

Reply:
Hi, jinzhe,

Thanks for updating the Slurm.py script.

Can I now use this new feature in dpgen by updating it in my computer?
Or I need to wait for a while until the next version is released?

Thanks again for your help.
Liang

Reply:
@njzjz I think `list` could be better.  It may be more clear.( I am not sure)  
It would be bettle if the user know clearly know the slurm scripte and this line looks like when they write the machine.json file

Reply:
@LiangMD-BGI you can update in your computer, or install this branch if you know how to use `git`. https://github.com/njzjz/dpgen/tree/custom_flags

Reply:
Hi, Jinzhe,

Thanks for your reply. I am not familair with git. 
Should I use: "pip install git+ssh://git@github.com/njzjz/dpgen/tree/custom_flags"?

Best,
Liang

Reply:
This should work:
```
pip install git+https://github.com/njzjz/dpgen#custom_flags
```